src/Color.js

```diff
@@ -0,0 +1,319 @@
+/**
+ * @file Color
+ * @author zhangjingyuan(zhangjingyuan02@baidu.com)
+ */
+
+import React, {PropTypes} from 'react';
+import InputComponent from './InputComponent';
+import ColorBox from './color/box';
+import Icon  from './Icon';
+import Mask from './Mask';
+import Button from './Button';
+import {create} from './common/util/cxBuilder';
+
+const cx = create('Color');
+const HEXCH = ['0', '1', '2', '3', '4', '5', '6', '7', '8', '9', 'A', 'B', 'C', 'D', 'E', 'F'];
+const colorBox = [
+        {color: 'FF425E', hue: 'FF0059'},
+        {color: 'C6EDE8', hue: '00FFF9'},
+        {color: '66CCCC', hue: '00FFF9'},
+        {color: 'FFFFFF', hue: '00FFF9'},
+        {color: '000000', hue: '0079FF'},
+        {color: 'CCCCCC', hue: '00CCFF'},
+        {color: '576069', hue: 'FF0026'},
+        {color: 'E2D3AC', hue: 'FFC600'},
+        {color: 'FDDA04', hue: 'FFDF00'},
+        {color: 'E58308', hue: 'FF8C00'},
+        {color: 'FF99CC', hue: 'FF00BF'},
+        {color: 'FC9D9B', hue: 'FF0000'},
+        {color: '59453E', hue: 'FF0000'},
+        {color: 'CCFF00', hue: 'B3FF00'}
+    ];
+
+export default class Color extends InputComponent {
+    constructor(props) {
+        super(props);
+
+        this.onSubmit = this.onSubmit.bind(this);
+        this.onHueChange = this.onHueChange.bind(this);
+        this.onSaturationChange = this.onSaturationChange.bind(this);
+        this.ondemoChange = this.ondemoChange.bind(this);
+        this.onLabelClick = this.onLabelClick.bind(this);
+        this.onHexChange = this.onHexChange.bind(this);
+        this.state = {
+            satValue: '00FFE7',
+            value: '00FFE7',
+            open: false,
+            color: '#FFFFFF',
+            rgb: [0, 255, 231]
+        };
```

Comment:
const value = this.state.value;
this.state = {
       ...this.state.value,
       satValue: value,
       open: false,
       color: XX.toHex(value),
       rgb: XX.toRgb(value)
};

//.....

Color.defaultProps = {
       ...InputComponent.defaultProps,
       defaultValue: 00FFE7
};